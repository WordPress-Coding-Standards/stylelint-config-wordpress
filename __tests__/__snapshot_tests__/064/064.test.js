'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '064', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '064' )],
		configFile: systemTestUtils.caseConfig( '064' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
