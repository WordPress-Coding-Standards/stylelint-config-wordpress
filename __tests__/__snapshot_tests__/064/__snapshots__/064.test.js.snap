// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`064 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "064/stylesheet.scss",
    "warnings": Array [
      Object {
        "column": 41,
        "line": 234,
        "rule": "block-no-empty",
        "severity": "error",
        "text": "Unexpected empty block (block-no-empty)",
      },
    ],
  },
]
`;
